packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -292,7 +293,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
           />
         </div>
 
-        { isOpen && (
+        { isOpen || (alwaysDisplaySuggestions && focused) && (
```

Comment:
There's a lot of logic in this components that differs depending on the isOpen state. I feel like with this change, you will be able to get a comboBox into a weird in between state where the behavior is inconsistent depending on how the menu was opened.

I think it would be cleaner if isOpen is the only thing that dictates if the menu is open... For example, by creating a simple function that looks at the  isOpen and the alwaysDisplaySuggestions props to determine if the menu is open

Reply:
Also, there's no way to close the menu when it's focused and alwaysDisplaySuggestions is true, is this the expected behavior?

On top of that, how will the user know that the value they selected is submitted? If they type or arrow around the menu options and press ENTER, they will get no indication that the value has been submitted

Reply:
Yep, these are all problems with this approach, I will go back and fix them
The behavior I want to enable is that when you focus the text box, the suggestions display, and when you change the freeform input, the suggestions display. 

I realize now forcing the state is not the correct move here. I will instead toggle isOpen state